mobile/src/reducers/calendar/calendar-events.reducer.ts

```diff
@@ -66,27 +67,47 @@ export const calendarEventsReducer: Reducer<CalendarEventsState> = (state = init
                 ...state,
                 weeks: weeks
             };
+        case 'CLAIM-SICK-LEAVE':
+            const claimDialog: DialogActiveState = {
```

Comment:
i think this configuration things can be moved to separate constant in a separate file? To separate constants definitely.